copier.yml

```diff
@@ -1,26 +1,38 @@
 # questions
+project_short_description:
+  type: str
+  help: "A short description of the project"
+  default: "Python package for "
+
+has_cli:
+  type: bool
+  help: "Does the project have a CLI?"
+  default: no
+
+is_django_package:
+  type: bool
+  help: "Is the project a Django package?"
+  default: no
+
 full_name:
   type: str
   help: "What's your name?"
+  default: '{{ ("exec(''from subprocess import run; import json; j = json.loads(run([\"gh\",\"api\",\"user\"], capture_output=True).stdout)'') or j[''name''] or j[''login'']" | eval) | env("PYPACKAGE_TEMPLATE_NAME") }}'
 
 email:
   type: str
   help: "Email address"
-  placeholder: "first.last@example.com"
+  default: '{{ ("exec(''from subprocess import run; import json; j = json.loads(run([\"gh\",\"api\",\"user/emails\"], capture_output=True).stdout)'') or [k for k in j if k[''primary'']][0][''email'']" | eval) | env("PYPACKAGE_TEMPLATE_EMAIL") }}'
```

Comment:
What happens if the person doesn't have `gh` installed? I think it's fully optional right now, but that seem to make it required.